recipes/rsgislib/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "3.2.0" %}
+
+package:
+  name: rsgislib
+  version: {{ version }}
+
+source:
+  fn: rsgislib-{{ version }}.tar.gz
+  url: https://bitbucket.org/petebunting/rsgislib/downloads/rsgislib-{{ version }}.tar.gz
+  sha256: a9a75b863556bbd055fbe5b774b63a86b73e33fb9fc45f01b32af6a8e21c65a4
+
+build:
+    number: 0
+    # CGAL etc do not support vc9
+    skip: True  # [py27 and win]
+    features:
+        - vc9  # [win and py27]
+        - vc14  # [win and py>=35]
```

Comment:
Thanks @ocefpaf for merging. BTW this package will not build with anything less than `vc14` on Windows anyway due to `mpfr`, but will remember for future.